mchf-eclipse/.settings/language.settings.xml

```diff
@@ -5,7 +5,7 @@
 			<provider copy-of="extension" id="org.eclipse.cdt.ui.UserLanguageSettingsProvider"/>
 			<provider-reference id="org.eclipse.cdt.core.ReferencedProjectsLanguageSettingsProvider" ref="shared-provider"/>
 			<provider-reference id="org.eclipse.cdt.managedbuilder.core.MBSLanguageSettingsProvider" ref="shared-provider"/>
-			<provider class="org.eclipse.cdt.managedbuilder.language.settings.providers.GCCBuiltinSpecsDetector" console="false" env-hash="1885161144660037189" id="ilg.gnuarmeclipse.managedbuild.cross.GCCBuiltinSpecsDetector" keep-relative-paths="false" name="CDT GCC Built-in Compiler Settings Cross ARM" parameter="${COMMAND} ${FLAGS} ${cross_toolchain_flags} -E -P -v -dD &quot;${INPUTS}&quot;" prefer-non-shared="true">
```

Comment:
thanks for that! I changed my setup and hope it has been the last time to spoil the commits ;-)!
Hmmm, I just wanted to add the commits "new filters 2k7, 2k9, 3k6" and "fixed non-working switchoff of filters". I made a rebase before this afternoon, so everything should be as the latest version of Andreas. After that I added those two commits regarding filters.
Andreas, let me know whether that worked or if I need to alter my setup again.